Readme.md

```diff
@@ -5,6 +5,10 @@
 
 Rewritten from [icefox/git-hooks](https://github.com/icefox/git-hooks), with extra features
 
+## Supported Go versions
+
+git-hooks supports the latest two Go version. (Currently 1.12 and 1.13)
```

Comment:
This is actually implementation details right? final user, which will use the binary directly do not care about which go version we use for development.

Reply:
Users may want to install git-hooks via `go get`, so I guess it's better to keep it.